README.md

```diff
@@ -1,9 +1,12 @@
-# dwyl hq
-:red_circle: open kimono
+# dwyl
 
-# Why?
+DWYL LTD is incorporated and registered in England and Wales with company number 09525434 and registered office is at DWYL LTD, 104 Mary Datchelor Close, London, SE5 7DY, United Kingdom.
```

Comment:
I think this needs a *tiny* bit of expansion, I always talk about first and foremost being an open source community and that is the friendliness I'd like to convey, see: https://github.com/dwyl/start-here/issues/117

I think that that and the mission is way *more important* than the registered company for now, particularly with regards to our github audience (vs our website audience which is more client-focused) and should be the first thing people read :blush: